sync/src/test/java/com/toggl/sync/sequence/ResolveOutstandingPushRequestTest.kt

```diff
@@ -0,0 +1,156 @@
+package com.toggl.sync.sequence
+
+import com.toggl.api.clients.SyncApiClient
+import com.toggl.api.exceptions.BadRequestException
+import com.toggl.api.exceptions.CallTimeoutException
+import com.toggl.api.exceptions.ForbiddenException
+import com.toggl.api.exceptions.InternalServerErrorException
+import com.toggl.api.exceptions.NotFoundException
+import com.toggl.api.exceptions.OfflineException
+import com.toggl.api.exceptions.UnauthorizedException
+import com.toggl.api.network.models.sync.PushResponse
+import com.toggl.common.services.PushRequestIdStorage
+import com.toggl.sync.common.CoroutineTest
+import com.toggl.sync.processing.ResponseProcessor
+import io.mockk.called
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.UUID
+import java.util.stream.Stream
+
+@DisplayName("The PushSync step")
```

Comment:
Probably © & 🍝 error?
```suggestion
@DisplayName("The ResolveOutstandingPushRequest step")
```

Reply:
You're right, thanks!